AdfDataflowFilePrettifier/regenerateTestFiles.sh

```diff
@@ -0,0 +1,3 @@
+cat AdfDataflowFilePrettifier.Tests/TestFiles/SampleDataFlow1_Ugly.json | ../AdfDataflowFilePrettifier.exe  -prettify -fromStdIn > AdfDataflowFilePrettifier.Tests/TestFiles/SampleDataFlow1_Pretty.txt

```

Comment:
My thinking was that if we're printing to stdOut then it feels more symmetric to read from stdIn, particularly if we're in a bash script. 👍 

But either way was fine, and it was actually easier to understand how to use -fromStdIn than -fromFile because the program is very strict about the order of command line args haha.

Happy to change if you'd rather

Reply:
I'm not particularly fussed :)